src/createServer.js

```diff
@@ -1,3 +1,43 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { validation } = require('./validation');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const splitUrl = req.url.split('?');
+
+    const path = splitUrl[0];
+    const textToConvert = path.slice(1);
+
+    const queryString = splitUrl[1];
+    const params = new URLSearchParams(queryString);
+
+    const toCase = params.get('toCase');
```

Comment:
Move it to the separate function
```suggestion
    const [..., ...] = ...( ... );
    
    OR
    
    const { ..., ... } = ...( ... );
```